src/docker-images/azure-blob-adapter/prebuild.sh

```diff
@@ -0,0 +1 @@
+cp -a ../../../../azure-blob-adapter/* ./
```

Comment:
Why not just put content in docker-images?

Reply:
I see, just copy from what other services do

Reply:
Can I do soft link for the directory?

Reply:
Moved